src/app/main/component/places/places.component.ts

```diff
@@ -153,10 +155,14 @@ export class PlacesComponent implements OnInit {
     }
   }
 
-  private bindLang(lang: string): void {
+  public bindLang(lang: string): void {
     this.translate.setDefaultLang(lang);
   }
 
+  private subscribeToLangChange(): void {
+    this.langChangeSub = this.localStorageService.languageSubject.subscribe(this.bindLang.bind(this));
```

Comment:
Need to use **takeUntil** rxjs operator in **pipe** or **unsubscribe** from this method.

Reply:
Thank you! Changes are edded.